perllib/FixMyStreet/Map/FMS.pm

```diff
@@ -12,7 +12,7 @@ use strict;
 sub map_template { 'fms' }
 
 sub map_javascript { [
-    '/vendor/OpenLayers/OpenLayers.fixmystreet.js',
+    '/vendor/OpenLayers/OpenLayers.wfs.js',
```

Comment:
Not sure there's any way round this, but this is a 40% increase in gzipped filesize :-/

Reply:
well, some faff means this is reduced to a 30 ish % increase.

Reply:
Ah, sadly FramedCloud is used by the fault assets stuff.